src/Utilities/addToPlaylistHandler.js

```diff
@@ -0,0 +1,25 @@
+
+function checkIfPresentInPlaylist(playlistsArray,playlist,videoDetails){
+ let selectedPlaylist = playlistsArray.find(item=>item.listId===playlist.listId)
+ 
+console.log("logging listVideos array...",selectedPlaylist)
+const videoObject= selectedPlaylist?.listVideos.find(video=>video.id===videoDetails.id)
+console.log({videoObject});
+ return videoObject;
+}
+
+
+export function addToPlaylistHandler(playlists,dispatch,playlist,videoDetails){
+  
+  const result = checkIfPresentInPlaylist(playlists,playlist,videoDetails);
+  if(!result){

```

Comment:
Here, you can remove the "result" and move the function to inside the if condition.

Reply:
yes we can